content/en/blog/2024/spring-starter-stable/index.md

```diff
@@ -0,0 +1,692 @@
+---
+title: The OpenTelemetry Spring Boot starter is now stable
+linkTitle: Spring Starter stable
+date: 2024-09-30
+author: >
+  [Gregor Zeitlinger](https://github.com/zeitlinger) (Grafana Labs), [Jean
+  Bisutti](https://github.com/jeanbisutti) (Microsoft)
+issue: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/11581
+sig: Java
+cSpell:ignore: Bisutti Gregor Zeitlinger
+---
+
+<!-- markdownlint-disable blanks-around-fences -->
+<?code-excerpt path-base="examples/java/spring-starter"?>
```

Comment:
```suggestion
<!-- markdownlint-disable blanks-around-fences -->
<?code-excerpt path-base="examples/java/spring-starter"?>
```

Let's not use code-excerpt in blog posts. They will outdated eventually but this is by design